News/NewsManager.php

```diff
@@ -91,6 +92,20 @@ protected function bind(NewsItem $entity, array $data)
     {
         $entity->setTitle($data['title']);
         $entity->setContent($this->getValue($data, 'content'));
+        $entity->setMediaDisplayOption($this->getMediaValue($data, 'displayOption', null));
+        $entity->setMedias(
+            array_map(
+                function ($id) {
+                    return $this->entityManager->getReference(Media::class, $id);
+                },
+                $this->getMediaValue($data, 'ids', [])
+            )
+        );
```

Comment:
the content of the media-property will be devided by this function into the `displayOption` and the media relation.